### DISCUSSION TITLE
Hitting "start" removes all files from queue...?

### DISCUSSION THREAD

**Discussion by mang0** [yermak/AudioBookConverter#233] [Q&A]
Hi there,

I think this is exactly what I'm looking for - I've got a book in a folder, around about 40 MP3 files, each file is one chapter. I would like to combine them all into an M4B file.

However, when I import the files and then hit 'start', they all seem to disappear from this list and nothing happens.

I'm running Win10. I tried both the portable version and the 'all users' MSI, both have the same issue.

I feel like I'm probably missing something. Any support greatly appreciated! Thanks for creating this software.

**Comment by yermak** [comment]
Hi,
You could try to look into log file it is located in Application Data in user home. This will give me more info what went wrong.
... Or You could upload you books somewhere into cloud and share link with me by email mynickname at gmail.
.. Or you could try to add just one-two files to see if it works.

If your files are located somewhere on network drive, program need to parse them for meta info, this could take a while. It's better to place them on local drive. Adding a progress of parsing meta information is on my todo list.

Let me know if it works.

**Reply by mang0** [reply]
Hi,

Thanks for responding so quickly. Much appreciated, and that's a very kind offer of you to do that for me.

Here's the log from Appdata:

https://pastebin.pl/view/43e23ec6

The files are stored on my C: drive, so I don't think that would be a problem - but good to know for the future, as if I can get this to work I would have various other books to convert too.

I did try with two files and even one file, and I was getting the same result.

I hope the log is useful. Thanks again for your time.

**Reply by mang0** [reply]
Sorry for the second quick message - it seems to be path issues. I see the file 'path.properties', and in it are listed

> ffmpeg=app/external/ffmpeg.exe
> ffprobe=app/external/ffprobe.exe
> mp4art=app/external/mp4art.exe
> mp4info=app/external/mp4info.exe

I double checked that ffmpeg and ffprobe executables are in those folders, and they are. I'm wondering if this is some issue with read/write properties somehow?

Running AudioBook Converter as admin still has the same errors though!

**Reply by yermak** [reply]
Please download previous version

**Reply by yermak** [reply]
Thanks for reporting. I broke pathes in latest release. On my dev pc it picked files from path env variable

**Reply by mang0** [reply]
Hi Yermak,

Your new update 5.5.17 fixes the env paths issues perfectly. All working for me now - thank you very much!

**Reply by yermak** [reply]
Thanks again for reporting. Was fixing it for linux and did not test build properly on windows. Such a shame :)
